### DISCUSSION TITLE
Production tax credit for CCS

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#94] [Q&A: Analysis]
Good afternoon! I have a question on the objective function for the calculation of the production tax credit.

In objective function, the parameter ptc_unit_value(i,v,r,t) used to calculate the value of PTC. For the coal-CCS and gas-CCS representation in ptc_unit_value(i,v,r,t), does it refer to the section 45Q? 

If yes, could you explain more? For example: why the section 45Q credit is not represented in $/metric ton CO2; what is the relationship between this inputted PTC value and the amount of CO2 captured by CCS? 

Thanks!

**Comment by wesleyjcole** [comment]
Because ReEDS is typically used sequentially, there is no foresight for the future payments from the PTC or 45Q payments.  Thus, to get the correct investment incentive we calculate the present value of the expected PTC or 45Q value and apply that to the investment variable.

**Reply by wanyingw193** [reply]
Thank you! Just a few more question for 45Q payment. 

(1) Does the current PTC_unit_value.csv for 45Q payments represent only the credits of the newv(v) class?
(2) In incentives_annual.csv, how are the values in the ptc_value column calculated?

Thank you so much for your time and patience!

**Reply by wesleyjcole** [reply]
The PTC values only apply to newv classes.  Initv classes are for existing generators that have already been built, so they do not have an investment variable.

The calc_financial_inputs.py and support_funtions.py have the ptc_value calculations, so that's the best place to look into those.

**Reply by wanyingw193** [reply]
Thanks for your response! I didn't ask the second question in the right way. I was wondering how the ptc_value ($/MWh) in inputs/incentives/incentives_annual.csv is determined based on the section 45Q credit ($/metric ton).
 
The reformed question is 'How to convert the 45Q credit from $/metric ton to $/MWh?" I am trying some calculations to get the same number of ptc_value in incentives_annual.csv, but was not able to do so. The equations I am using is as follows.

ptc_value ($/MWh) = 45Q credits per year ($/metric ton) / capture rate (metric ton/MWh).
         
Where (1) the 45Q credits per year represent $50/metric ton of CO2 storage in saline reservoir, and (2) the capture rate (metric tons/MWh) is obtained from run/[case_name]/input_cases/rep_[case_name].gdx.

Did I miss something?
